structured_type_test.go

```diff
@@ -1621,3 +1623,135 @@ func forAllStructureTypeFormats(dbt *DBTest, f func(t *testing.T, format string)
 		})
 	}
 }
+
+func TestNullObject(t *testing.T) {
```

Comment:
Names are global here, so maybe we can have something a bit more specific? Maybe `TestSelectingNullObjectsInArrowBatches`?

Reply:
done